README.md

```diff
@@ -1,81 +1,158 @@
-  <li>Function that will count up to a number and back down and return a string of the climb
-  <br>
-    <code> countUpAndDown(3) // should return "1 2 3 2 1"</code>
-  </li>
-  <li>Write a function that will tell you all of the words in an array that contain the letter `a`
-  <br>
-    <code > wordsWithA(['cat', 'rabbit', 'dog', 'frog']) // should return ['cat', 'rabbit']</code>
-  </li>
-  <li>Write a function that will tell you all of the words in an array that contain a specified letter
-  <br>
-    <code > wordsWithLetter("g", ['cat', 'rabbit', 'dog', 'frog']) // should return ['dog', 'frog']</code>
-  </li>
-  <li>Function that returns the longest word in sentence
-  <br>
-    <code>longestWord("The cat in the house") // should return "house"</code>
-  </li>
-  <li>Function that returns the largest even number
-  <br>
-    <code>largestEvenNumber([1,2,3,10,4,7,0]) // should return "10"</code>
-  </li>
-</ol>
-
-## Extra Practice
-
-Create word guessing game where the user gets infinite tries to guess the word (like Hangman without the hangman, or like Wheel of Fortune without the wheel and fortune).
-
-- Create two global arrays: one to hold the letters of the word (e.g. `['F', 'O', 'X']`), and one to hold the current guessed letters (e.g. it would start with `['_', '_', '_]'` and end with `['F', 'O', 'X']`)`.
-- Write a function called guessLetter that will:
-  - Take one argument, the guessed letter.
-  - Iterate through the word letters and see if the guessed letter is in there.
-  - If the guessed letter matches a word letter, changed the guessed letters array to reflect that.
-  - When it's done iterating, it should log the current guessed letters ('F__') and congratulate the user if they found a new letter.
-  - It should also figure out if there are any more letters that need to be guessed, and if not, it should congratulate the user for winning the game.
-  - Pretend you don't know the word, and call guessLetter multiple times with various letters to check that your program works.
-
-```js
-// start of the game
-const wordLetters     = ['G', 'O', 'A', 'T'];
-const guessedLetters  = ['_', '_', '_', '_'];
-
-
-// playing the game
-guessLetter('G'); // "Correct, G _ _ _"
-guessLetter('I'); // "Incorrect, G _ _ _"
-guessLetter('O'); // "Correct, G O _ _"
-guessLetter('A'); // "Correct, G O A _"
-guessLetter('L'); // "Incorrect, G O A _"
-guessLetter('T'); // "You Win, G O A T"
-```
-**How To: Make it like Hangman:**
-- Keep track of all the guessed letters (right and wrong) and only let the user guess a letter once. If they guess a letter twice, do nothing.
-- Keep track of the state of the hangman as a number (starting at 0), and subtract or add to that number every time they make a wrong guess.
-- Once the number reaches 6 (a reasonable number of body parts for a hangman), inform the user that they lost and show a hangman on the log.
+1#  
+
+const summation = function(array)
+{
+    return array.reduce(function(a,b){return a+b},0);
```

Comment:
The function `reduce()` works on arrays, while the parameter in this function is expecting a number. so this wont work.